### PR TITLE
[WIP] Arabic translation

### PR DESCRIPTION
**Pull request by shadeed**
# Translation Progress
## Basic
- [ ] `_data/translations.yml`
- [ ] `_data/languages.yml`
## Code (comments only)
- [ ] `_includes/snippets`
## Readme
- [x] `README.md`
## Content
- [ ] `_architecture.md`
- [ ] `_author.md`
- [ ] `_comments.md`
- [ ] `_conditions.md`
- [ ] `_contributing.md`
- [ ] `_errors.md`
- [ ] `_extend.md`
- [ ] `_introduction.md`
- [ ] `_loops.md`
- [ ] `_mixins.md`
- [ ] `_naming.md`
- [ ] `_rwd.md`
- [ ] `_sass.md`
- [ ] `_syntax.md`
- [ ] `_tldr.md`
- [ ] `_toc.md`
- [ ] `_tools.md`
- [ ] `_variables.md`

### CODE REVIEW

File: ar/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: default
+language: ar
+direction: rtl
+---
+
+{% include chapters.html path = "/en/" %}
```
Comment (HugoGiraudel) [inline ar/index.md]: Please change this into `{% include chapters.html %}` as stated in: https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/TRANSLATING.md.

**Comment by HugoGiraudel** [comment]
Hello. What is the current status? :)

**Comment by shadeed** [comment]
Been busy the last month in some projects but I promise that we will finish a big part of the translation by this month. :)  

--  
Ahmad Shadeed
Twitter: @shadeed9
www.ishadeed.com

Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by HugoGiraudel** [comment]
Hey there! Any news regarding this? :)

**Comment by msjaber** [comment]
Hey Hugo!

I know it's been too long, and I apologize for being late. But the thing is
everything would be translated and uploaded by the end of this Saturday!

You have my word! :)




On Sun, Jan 3, 2016 at 6:00 PM, Hugo Giraudel <notifications@github.com>
wrote:

> Hey there! Any news regarding this? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HugoGiraudel/sass-guidelines/pull/254#issuecomment-168510330>
> .
>



-- 





Mohammad S. Aljaber : @msaljaber <https://twitter.com/MSAljaber>.

**Comment by HugoGiraudel** [comment]
There is no rush, take all the time you need. I just wanted to know what is going on. :)

**Comment by HugoGiraudel** [comment]
Hey there. Little reminder that version 1.3 has been released! 🎉

You can either keep going on 1.2, then we merge, then you work on 1.3. Or go straight to 1.3 right away. It’s really up to you.

On topic, do you need me to find you any support on this? Or do you think you could find time and energy to finish it just the 2 of you? :)

**Comment by HugoGiraudel** [comment]
Hey @shadeed. Is there any progress on this? Are you willing to continue or shall I close it? :)

**Comment by shadeed** [comment]
@HugoGiraudel Hi Hugo! So sorry but in the meantime I won't be able to continue working on this. Hopefully in the near future I will get the chance to do so.  cc @msjaber

**Comment by HugoGiraudel** [comment]
Hi there! Shall I close this? :)

**Pull request closed without merging**